src/main/java/controller/BlackJackGame.java

```diff
@@ -1,4 +1,70 @@
 package controller;
 
+
+import domain.card.Deck;
+import domain.user.*;
+import view.InputView;
+import view.OutputView;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+import static java.util.stream.Collectors.toList;
+
 public class BlackJackGame {
+
+    public static void main(String[] args) {
+        Players players = inputPlayers();
+        Dealer dealer = new Dealer();
+        Deck deck = new Deck();
+        deck.distributeCardsToPlayersAndDealer(players, dealer);
+        OutputView.printDistributeCardsMessageAndCardsOfAllUsers(players, dealer);
+        proceedPlayersTurn(players, deck);
+        proceedDealerTurn(dealer, deck);
+        OutputView.printDealerAndPlayersResult(players, dealer);
+        OutputView.printDealerAndPlayersProfit(players, dealer);
+    }
+
+    private static Players inputPlayers() {
+        List<PlayerName> playerNames = inputPlayerNames();
+        List<Player> players = playerNames.stream()
+                .map(BlackJackGame::convertNamesAndBettingMoneyIntoPlayers)
+                .collect(toList());
+        return new Players(players);
+    }
+
+    private static List<PlayerName> inputPlayerNames() {
+        return InputView.inputNames().stream()
+                .map(String::trim)
+                .map(PlayerName::new)
+                .collect(toList());
+    }
+
+    private static Player convertNamesAndBettingMoneyIntoPlayers(PlayerName playerName) {
+        BettingMoney bettingMoney = convertAmountIntoBettingMoney(InputView.inputAmount(playerName));
+        return new Player(playerName, bettingMoney);
+    }
+
+    private static BettingMoney convertAmountIntoBettingMoney(BigDecimal amount) {
+        return new BettingMoney(amount);
+    }
+
+    private static void proceedPlayersTurn(Players players, Deck deck) {
+        players.getPlayers().forEach(player -> proceedPlayerTurn(player, deck));
```

Comment:
한 줄로 여러 메서드를 연속해서 호출하는 것 보다 아래와 같이 줄 바꿈 해주는 것이 좋습니다 😃
```suggestion
        players.getPlayers()
        .forEach(player -> proceedPlayerTurn(player, deck));
```